### ISSUE TITLE
10.25

### ISSUE THREAD

**Issue by shakingkelly** [AprilSuna/TwitterDashboard#31]
1. Set up server on compute engine (shut down when not running)
2. [BERT language model (Kaggle kernel, torch version)](https://www.kaggle.com/yuval6967/toxic-train-bert-base-pytorch)
3. Network features (API search, first 5000)
4. User description feature
5. Add submit button for each user
6. Add user profile (handle, profile image, description etc) to labeling page

**Comment by shakingkelly** [comment]
[Test gcloud compute engine](https://github.com/GoogleCloudPlatform/reliable-task-scheduling-compute-engine-sample)

**Comment by AprilSuna** [comment]
[GCP NVIDIA Driver](https://www.jianshu.com/p/7d91aaac6416)

**Comment by shakingkelly** [comment]
Instances: k1109 (tested cron executor, debugged python3 vs 2)

**Comment by shakingkelly** [comment]
[cloud scheduler (cron)](https://github.com/GoogleCloudPlatform/reliable-task-scheduling-compute-engine-sample)

**Comment by AprilSuna** [comment]
corn job tutorial! so nice!
https://cloud.google.com/scheduler/docs/start-and-stop-compute-engine-instances-on-a-schedule#set_up_the_jobs_to_call
